src/api/Feed.js

```diff
@@ -181,7 +181,11 @@ export const getParsedFileActivitiesResponse = (response?: { entries: FileActivi
                     return annotationItem;
                 }
                 case FILE_ACTIVITY_TYPE_APP_ACTIVITY: {
-                    return { ...source[FILE_ACTIVITY_TYPE_APP_ACTIVITY] };
+                    const appActivityItem = { ...source[FILE_ACTIVITY_TYPE_APP_ACTIVITY] };
+
+                    appActivityItem.created_at = appActivityItem.occurred_at;
```

Comment:
In the API schema, it looks like the schema shows a created_at and an occurred_at property. What is the reasoning behind setting created_at and occurred_at to the same value? Does one of them not have a value?

Reply:
created_at does not have a value in file_activities